ui/shared/DropdownMenuItems.jsx

```diff
@@ -31,19 +31,13 @@ const DropdownMenuItems = ({
   <DropdownMenu className="overflow-auto dropdown-menu-height">
     {/* Items pinned to top of dropdown */}
     {pinned.length > 0 &&
```

Comment:
We wrote the `pinned.length` condition 2 times.

Could you make only one condition out of it & then return the `pinned.map()` and the `<DropdownItem divider />`?